modules/remove.js

```diff
@@ -0,0 +1,6 @@
+const remove = (arry, re) => {
+  arry.delbook(parseInt(re.dataset.id, 10));
+  localStorage.setItem('array', JSON.stringify(arry));
+  re.parentNode.remove();
+};
+export default remove;
```

Comment:
- [ ] Good job on fixing the `add` case. Kindly fix the same problem requested by the previous reviewer in the `delete` case. The problem happens because first the data is stored in a key called `array` in the local storage array of key `array`
![image](https://user-images.githubusercontent.com/24703171/212666333-1c731d5c-db39-44fa-b37d-a923fe76f07b.png)
but after the first delete the data is converted to an object containing one key `lists` with key `array`
![image](https://user-images.githubusercontent.com/24703171/212667003-13654e29-6007-46b7-93b2-77f20f780edb.png)
so after I refresh the page after the delete process I get this error 
![image](https://user-images.githubusercontent.com/24703171/212667169-b3cb7f9f-3ba4-488e-8a0e-4ee2e7209e15.png)
and the data isn't loaded.